src/internal/pfsdb/common.go

```diff
@@ -114,3 +114,28 @@ func IsNotFoundError(err error) bool {
 		errors.As(err, &CommitNotFoundError{}) ||
 		errors.As(err, &BranchNotFoundError{})
 }
+
+// GraphOpt is used to configure depth and limit parameters for provenance and subvenance queries.
```

Comment:
I'm not really a fan of this configuration pattern. With functional options (https://github.com/tmrts/go-patterns/blob/master/idiom/functional-options.md) you don't need to worry about unset configurations because they will be unchanged from their default value. Also, I think just `depth` and `limit` as the names seems better.

Reply:
Thanks for linking that pattern, seems like a much better alternative.